src/modules/apps/e2e/index.tsx

```diff
@@ -29,7 +29,7 @@ class EndToEndCommand {
     const { data: workspaceAppsList } = await apps.listApps()
     const appItem = workspaceAppsList.find(({ app }) => app.startsWith(cleanAppId))
 
-    if (appItem.id === undefined) {
+    if (appItem?.id === undefined) {
```

Comment:
_nit (optional)_: Could you change it to
```ts
if (appItem === undefined) {
```
It avoids the extra conditional and won't lead others (like myself just now 😂) to think that `id` might be undefined - I checked and it's always the same as `appItem.app`.

Reply:
Cool, I just assumed that `id` could be undefined. Will update!